sdk/search/Azure.Search.Documents/src/Models/SynonymMap.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.Search.Documents.Models
+{
+    [CodeGenSchema("SynonymMap")]
+    public partial class SynonymMap
```

Comment:
We should check with Anne to see if there are other Cognitive services that expose synonym mapping and possibly rename this `SearchSynonym` or similar.

Reply:
So far, none do but we reached out to the cognitive services PM. Will track separately based on what we hear back.